DataFormats/Scouting/test/TestReadRun3Scouting.cc

```diff
@@ -1079,6 +1079,15 @@ namespace edmtest {
       if (vertex.isValidVtx() != static_cast<bool>((expectedVertexIntegralValues_[2] + iOffset) % 2)) {
         throwWithMessage("analyzeVertexes, isValidVtx does not equal expected value");
       }
+      if (vertex.xyCov() != expectedVertexFloatingPointValues_[7] + offset) {
+        throwWithMessage("analyzeVertexes, xy cov. does not equal expected value");
+      }
+      if (vertex.xzCov() != expectedVertexFloatingPointValues_[8] + offset) {
+        throwWithMessage("analyzeVertexes, xz cov. does not equal expected value");
+      }
+      if (vertex.yzCov() != expectedVertexFloatingPointValues_[9] + offset) {
+        throwWithMessage("analyzeVertexes, yz cov. does not equal expected value");
+      }
```

Comment:
I'd expect these checks to fail for the test files written before this PR. I believe the reader needs the expected vertex class version to be passed in.

(but let's test anyway)

Reply:
@mmasciov this commit https://github.com/cms-sw/cmssw/commit/d1da825b28d73f408a575cd5d04189707e6a0393 on top of your branch will allow the unit tests to pass.

Reply:
Done. Thanks, @mmusich!